### PR TITLE
KYAN-147 typed title component

### PR DESCRIPTION
**Pull request by martyna-ds**
## Description
Expand title component with animated endings - typing effect - provided with typed.js plugin

## Motivation and Context
business need

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html [hunk 1]
```diff
@@ -24,6 +24,18 @@ <h6 class="has-text-primary is-1 eyebrow">
   </sly>
   <h1 id="${model.anchorId}" style="text-align: ${model.titleAlign @ context='styleToken'};" data-sly-element="${model.element}" class="${model.titleClasses @ join=' '}">
     ${model.textAsOneLine @ context='html'}
+    <sly data-sly-test="${model.addAnimatedEndings}">
+      <span id="typed-strings" class="${model.endingsClasses @ join=' '}">
```
Comment (paweljozwicki) [inline applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html]: Should it be an id? For more than one title with an animated endings we would have multiple elements with the same id.

File: applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html [hunk 2]
```diff
@@ -24,6 +24,18 @@ <h6 class="has-text-primary is-1 eyebrow">
...
+        <sly data-sly-list.item="${model.endings}">
+          <p>
```
Comment (paweljozwicki) [inline applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html]: Should we use a paragraph inside the heading? Its direct parent is a span element, but I still don't know if it's correct.

File: applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html (hunk 1 above)
Comment (martyna-ds) [reply applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html]: right, I will change it to class

File: applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html (hunk 2 above)
Comment (martyna-ds) [reply applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html]: yeah, it is not correct, changed to `span`

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**